python/llm/src/ipex_llm/transformers/models/chatglm2.py

```diff
@@ -300,6 +303,8 @@ def chatglm2_attention_forward(
     attn_weights = None
     if use_sdp(q_len, kv_seq_len, head_dim, query_states):
         import xe_addons
+        if use_compresskv and attention_mask is not None:
+            attention_mask = None
```

Comment:
Do you want to set it to None here?

Reply:
Yes, the attn_mask of chatglm2 is different from other models in lookahead, and needs to be fixed in other pr.